openapi.yaml

```diff
@@ -749,7 +749,7 @@ paths:
       security:
       - personalAccessToken: []
       - oauth:
-        - events:read_only
+        - events:read_write
```

Comment:
I'm not sure this is correct. The blueprint shows this as read only:

https://bits.linode.com/LinodeAPI/apinext/blob/development/apinext/blueprints/account/events.py#L287

What prompted this change?

Reply:
Two things: It's on a POST and it's x-linode-grant is read_write.

Reply:
The code and history have this endpoint as read only even before we started our blueprint refactor. Josh, is this a bug on our end?

Reply:
This endpoint, although a POST, has read_only scopes because it allows clients to mark events as seen without giving the clients read_write account permissions, which opens things up to allow a client to do A LOT more than just marking an event as seen.

Reply:
Thanks for the discussion here -- the x-linode-grant should also be listed as read only here. Updated the spec accordingly.